bats/fb-katello-content.bats

```diff
@@ -89,6 +106,21 @@ setup() {
     --path="/tmp/stbenjam-dummy-0.2.0.tar.gz" | grep -q "Successfully uploaded"
 }
 
+@test "upload ostree_ref" {
+  if ! tIsRHEL 8; then
+    skip "OSTree content is only applicable on EL 8 systems"
+  fi
+
+  tSkipIfOlderThan43
+
+  wget --no-parent -r https://fixtures.pulpproject.org/ostree/small/
+  tFileExists fixtures.pulpproject.org/ostree && \
+    tar --exclude="index.html" -cvf "fixtures_small_repo.tar" -C fixtures.pulpproject.org/ostree "small"
```

Comment:
Just wondering, why do you need `&&` here? bats will stop if an assertion is false so this should also work:
```suggestion
  tFileExists fixtures.pulpproject.org/ostree
  tar --exclude="index.html" -cvf "fixtures_small_repo.tar" -C fixtures.pulpproject.org/ostree "small"
```